checkout-v3/modules-sdks/mobile-sdk/index.md

```diff
@@ -16,34 +16,36 @@ UIViewController on iOS) that you can integrate in your mobile application in
 the usual fashion. To work, these components need data from the Swedbank Pay
 APIs, which you must retrieve through your own servers. At the core, the
 libraries are agnostic as to how the communication between your app and your
-servers happens, but an implementation is provided for a server that implements
-what we call the Merchant Backend API. The Merchant Backend API is designed to
-transparently reflect the Swedbank Pay API, and the data types used to configure
-the mobile libraries allow you to organically discover the capabilities of the
-system.
+servers happens, but an example implementation is provided for a server that
+implements what we call the Merchant Backend API. The Merchant Backend API is
+designed to transparently reflect the Swedbank Pay API, and the data types used
+to configure the mobile libraries allow you to organically discover the
+capabilities of the system.
 
-The SDK is designed to integrate the Swedbank Pay UI inside your application's
-native UI. It generates any html pages required to show the Swedbank Pay UI
-internally; it does not support using a Checkout or Payments web page that you
-host yourself. If doing the latter fits your case better, you can show your web
-page in a Web View instead. In that case, you may benefit from the [collection
-of information about showing Checkout or Payments in a Web View][plain-webview].
+The SDK is designed to integrate the Swedbank Pay Seamless View Payment UI
+inside your application's native UI. It generates any html pages required to
+show the Swedbank Pay UI internally; it does not support using a Checkout or
+Payments web page that you host yourself. If doing the latter fits your case
+better, you can show your web page in a Web View instead. In that case, you may
+benefit from the [collection of information about showing Checkout or Payments
+in a Web View][plain-webview].
 
 ## Prerequisites
 
 To start integrating the Swedbank Pay Mobile SDK, you need the following:
 
-*   An [HTTPS][https] enabled web server.
 *   An agreement that includes [Swedbank Pay Digital Payments][checkout],
     specifically [Enterprise][checkout-enterprise] or [Payments
     Only][checkout-payments-only].
```

Comment:
We have discontinued and renamed a couple of things here, so we have to do a bit of a cleanup :) Enterprise isn't offered anymore, so we can remove that link. The documentation is hidden from view, but present as I believe we still have a merchant running on it.

Perhaps keep it to "An agreement that includes Swedbank Pay Digital Payments". They are supposed to be going up on version 3.1, but I doubt that is specified in the agreement :)